src/pages/register/register.js

```diff
@@ -51,22 +50,11 @@ function Register() {
   const handleRegister = async () => {
     setShouldValidate(true);
     if (allFieldsValidated) {
-      delete user.confirmPassword;
+      setToLocalStorage('accessToken', null);
```

Comment:
Why do we need this line? I don't think that we are able to go to the registration page if we have a token

Reply:
it might solve partially an issue where our token gets expired, so it would remove the expired token.